_stbt/control_gpl.py

```diff
@@ -162,6 +163,14 @@ def press(self, key):
         from .control import UnknownKeyError
         keycode = self._KEYNAMES.get(key)
         if keycode is None:
+            if isinstance(key, int):
+                keycode = key
+            elif re.match(r"^[0-9]+$", key):
+                keycode = int(key, base=10)
+            elif re.match(r"^0[xX][0-9a-fA-F]+$", key):
+                keycode = int(key, base=16)
```

Comment:
Python can do this working out itself.  Perhaps something like:

    try:
        keycode = int(key, 0)
    except ValueError:
        pass

would be better.  But maybe not, `015 == 13`  may be a little confusing.